zebra/rtadv.c

```diff
@@ -996,6 +1019,51 @@ void zebra_interface_radv_enable(ZAPI_HANDLER_ARGS)
 	zebra_interface_radv_set(client, hdr, msg, zvrf, 1);
 }
 
+DEFUN (ipv6_nd_ra_fast_retrans,
```

Comment:
I thought there was some desire to prefer `DEFPY` with new clis (I know there aren't lots of args to this one; just asking the general question)

Reply:
I hadn't heard that if there weren't particular gains to be made with DEFPY.  Like above, I just copied the existing form of the commands in this file.  If preferred, I can make the changes for this command.